modvendor/modvendor_test.go

```diff
@@ -928,20 +1005,21 @@ func TestModVendorWithRef(t *testing.T) {
 
 	const vendordir = "/vendor"
 	got := modvendor.Vendor(rootdir, vendordir, source)
+	vendoredAt := modvendor.Dir(vendordir, source)
 	assertVendorReport(t, modvendor.Report{
 		Vendored: map[string]modvendor.Vendored{
-			source.Raw: {
+			vendoredAt: {
 				Source: source,
 				Dir:    modvendor.Dir(vendordir, source),
 			},
 		},
 	}, got)
 
-	cloneDir := got.Vendored[source.Raw].Dir
+	cloneDir := got.Vendored[vendoredAt].Dir
```

Comment:
`cloneDir == vendoredAt` but maybe makes sense to get the dir from the report just to be sure?

Reply:
hmm yeah getting from the report validates that it came correctly on the report